model-optimizer/extensions/front/mxnet/arange_like_replacer.py

```diff
@@ -0,0 +1,80 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.ops.elementwise import Add
+from extensions.ops.gather import Gather
+from extensions.ops.range import Range
+from mo.front.common.partial_infer.utils import int64_array
+from mo.front.common.replacement import FrontReplacementOp
+from mo.front.tf.graph_utils import create_op_with_const_inputs
+from mo.graph.graph import Graph
+from mo.ops.shape import Shape
+from mo.ops.reshape import Reshape
+from mo.ops.squeeze import Squeeze
+from mo.utils.error import Error
+
+
+class ArangeLikeReplacer(FrontReplacementOp):
+    op = 'arange_like'
+    enabled = True
+
+    def replace_sub_graph(self, graph: Graph, match: dict):
+        node = match['op']
+        name = node.soft_get('name', node.id)
+        if node.repeat != 1:
+            raise Error("arange_like node {} with non default repeat value {} "
+                        "is not supported".format(name, node.repeat))
+        axis = node.axis
+        shape_node = Shape(graph, {'name': name + '/shape'}).create_node()
+        range_node = create_op_with_const_inputs(graph, Range, {0: int64_array(node.start),
+                                                                2: int64_array(1)}, {'name': name + '/Range'})
+        node.in_port(0).get_connection().set_destination(shape_node.in_port(0))
+
+        if axis:
```

Comment:
if axis is 0, which branch should work? In tests you check axis None and not None (3), should we check here that axis is not None?

Reply:
Done